libs/ledger-live-common/src/families/polkadot/bridge.integration.test.ts

```diff
@@ -1,7 +1,7 @@
 import "../../__tests__/test-helpers/setup";
 import { testBridge } from "../../__tests__/test-helpers/bridge";
-import { dataset } from "@ledgerhq/coin-polkadot/bridge.integration.test";
+import { dataset } from "@ledgerhq/coin-polkadot/test/bridge.integration.test";
 
-import "@ledgerhq/coin-polkadot/bridge.integration.test";
+import "@ledgerhq/coin-polkadot/test/bridge.integration.test";
```

Comment:
Does it have to be imported that way, or should we export everything we need from the file then import it explicitly, as done above?

Reply:
You mean that this import is useless? I will test it.
It is a legacy code that I didn't challenged.